source/jquery.flot.selection.js

```diff
@@ -447,8 +457,16 @@ The plugin allso adds the following methods to the plot object:
                 ctx.translate(plotOffset.left, plotOffset.top);
 
                 var c = $.color.parse(o.selection.color);
+                var v = o.selection.visualization;
```

Comment:
I'd like us to start moving away from these poorly named locals (the below 'sf' being a worse offender).

Reply:
I agree on moving away from poorly named local variables. However, I disagree on 'sf' (abbreviation  for scaling factor) being any worse than 'c' for 'color' or 'oX' for I don't even know what. 

If I have to make a decision on whether or not to follow existing style in the code even if it was poor in my opinion, I will always choose to follow it if I am not going to refactor the whole file or if I am not told otherwise.

That being said, as per your request, I will refactor the variables I introduced.